www/js/App.tsx

```diff
@@ -42,6 +43,16 @@ const defaultRoutes = (t) => [
 ];
 
 export const AppContext = createContext<any>({});
+const CUSTOM_LABEL_KEYS_IN_DATABASE = ['mode', 'purpose'];
+type CustomLabelMap = {
+  [k: string]: string[];
+};
+
+const scenes = {
+  label: withErrorBoundary(LabelTab),
+  metrics: withErrorBoundary(MetricsTab),
+  control: withErrorBoundary(ProfileSettings),
+};
```

Comment:
```suggestion
```

Duplicated code that can be removed.
I think this was also my mistake from bad merge conflict resolution